tests/resources/at-commands.mjs

```diff
@@ -1,443 +1,69 @@
 import * as keys from './keys.mjs';
 
-const KNOWN_ATS = {
-  jaws: 'JAWS',
-  nvda: 'NVDA',
-  voiceover: 'VoiceOver'
-}
+export class commandsAPI {

```

Comment:
Is the `commandsAPI` the pattern we are going to use in the production system?

Reply:
Right now, this file is part of the test suite, it's for rendering the test files. So only indirectly, because the production system consumes the tests

Reply:
In this case, can we use a more descriptive class name or have a JSDoc about what this class does? `commandsAPI` wasn't clear enough for me to understand until I saw it in use.

Reply:
good point. There should be documentation. I'll add it right now!